rules/rules.libsonnet

```diff
@@ -2,5 +2,4 @@
 (import 'apps.libsonnet') +
 (import 'kube_scheduler.libsonnet') +
 (import 'node.libsonnet') +
-(import 'kubelet.libsonnet') +
-(import 'windows.libsonnet')
+(import 'kubelet.libsonnet')
```

Comment:
This removes windows rules from the default build, see [this comment](https://github.com/kubernetes-monitoring/kubernetes-mixin/pull/1013#discussion_r1936151932) where I was previously concerned about this change.